src/styles/card.scss

```
@@ -0,0 +1,93 @@
+.card {
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+  background: $card-background-color;
+  box-sizing: border-box;
+
+  &__img {
+    margin-top: 33px;
+    margin-inline: 19px;
+    width: $card-img-width;
+    height: $card-img-height;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    margin-inline: 16px;
+    margin-top: 40px;
+    color: $card-title-color;
+  }
+
+  &__subtitle {
+    font-size: 10px;
+    line-height: 14px;
+    color: $card-subtitle-color;
+    margin-inline: 16px 84px;
+    margin-top: 4px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+    margin-inline: 16px;
+  }
+
+  &__review {
+    font-size: 10px;
+    line-height: 14px;
+    text-align: right;
+    color: $card-title-color;
+    margin: auto 0;
+  }
+
+  &__priceText {
```

```suggestion
  &__price-text {
```
follow bem naming conventions